src/main/java/org/jenkinsci/plugins/casc/DataBoundConfigurator.java

```diff
@@ -248,4 +246,9 @@ public String getDisplayName() {
         final Descriptor descriptor = getDescriptor();
         return descriptor != null ? descriptor.getDisplayName() : "";
     }
+
+    private static <T> T getDefaultValue(Class<T> clazz) {
+        // Recipe from: https://stackoverflow.com/a/38243203/636849
+        return (T) Array.get(Array.newInstance(clazz, 1), 0);
```

Comment:
see com.google.common.base.Defaults#defaultValue

Reply:
Thanks, I'm going to use this